jx_sqlite/expressions.py

```diff
@@ -29,7 +29,7 @@
 
 @extend(Variable)
 def to_sql(self, schema, not_null=False, boolean=False):
-    cols = [c for cname, cs in schema.items() if startswith_field(cname, self.var) for c in cs]
+    cols = [c for cname, cs in schema.items() if startswith_field(get_property_name(cname), get_property_name(self.var)) for c in cs]
```

Comment:
We are looking up names of variables by their path prefix.  We should not patch like this, rather improve what schema is giving us.